src/components/pages/home/HomePage.js

```diff
@@ -1,10 +1,16 @@
 import React, { Component } from 'react';
-import TempPage from '../temp';
```

Comment:
I think we can delete the whole temp folder now - nothing should be using it any more!